src/js/components/explorer/detail/UnreportedErrorScreen.jsx

```diff
@@ -25,6 +25,7 @@ export default class UnreportedErrorScreen extends React.Component {
                             <li>An agency has a submission deadline extension, e.g. <a href="https://usaspending-help.zendesk.com/hc/en-us/articles/360001249973-Spending-Explorer-Question-I-m-looking-for-data-from-the-Department-of-Defense-DoD-and-can-t-find-it-is-it-shown-here-">Department of Defense (DOD)</a></li>
                             <li>An agency is not required to submit certain data elements</li>
                             <li>There are several accounts that represent a small percentage of overall federal spending that do not report to Treasury, but are included in the President&#8217;s budget.</li>
+                            <li>Monthly submission data is still loading on the websitew</li>
```

Comment:
If this is the final copy, it needs a minor correction to "websitew"

Reply:
I think you're looking at the old stuff.